CHANGELOG.md

```diff
@@ -8,6 +8,11 @@ Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unrealeased]
 
+### Changed
```

Comment:
in your commit message:
intead of to retrieve them through > instead of retrieving them through